notebooks/0201_create-composite-history.py

```diff
@@ -239,12 +252,18 @@ class BiomassBurningOption(StrEnum):
 ).pix.assign(scenario=HISTORY_SCENARIO_NAME)
 wmo_2022
 
+# %%
+cmip_inversions = load_csv(
+    DATA_ROOT / "global" / "esgf" / "CR-CMIP-0-4-0" / f"inverse_emissions_{CMIP_CONCENTRATION_INVERSION_ID}.csv"
+).pix.assign(scenario=HISTORY_SCENARIO_NAME)
+
```

Comment:
Maybe I missed it, but I don't think it is described what these data are or how these should be obtained?

Reply:
Ooops no I can definitely make that clearer. Short answer: run `notebooks/0110_cmip-conc-inversions.py`